runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -10941,24 +10869,7 @@ J9::Z::TreeEvaluator::VMnewEvaluator(TR::Node * node, TR::CodeGenerator * cg)
 
```

Comment:
Are comments from line 100849-100866 relevant ?

Reply:
It does not match the current instruction selection. I remove it